docs/root/start/sandboxes/dynamic-configuration-filesystem.rst

```diff
@@ -100,6 +100,13 @@ You can do this using ``sed`` inside the container:
 
    docker-compose exec -T proxy sed -i s/service1/service2/ /var/lib/envoy/cds.yaml
 
+Notice that `sed -i` means replace not edit in place.
+You can create a new file and useing `mv` to do:
+
+.. code-block:: console
```

Comment:
this is unnecessary - and incorrect - if we were to add it we would need to also add to verify.sh - but rather just remove

Reply:
Removed.